plugin/src/main/java/io/jenkins/plugins/coverage/metrics/model/ChangedLinesModel.java

```diff
@@ -0,0 +1,57 @@
+package io.jenkins.plugins.coverage.metrics.model;
+
+import java.util.Objects;
+
+public class ChangedLinesModel {
+    private int startLine;
+    private int endLine;
+    private Type type;
+
+    public ChangedLinesModel(final int start_line, final int end_line, final Type type) {
+        this.setStartLine(start_line);
```

Comment:
We should not call other methods from a constructor that can be overwritten. This could unexpectedly change the constructor's behavior in subclass. Directly set the attributes and make them final. 
Hence, we do not need setters here,